ddr/lib/ddr-scanner/dwarf/DwarfScanner.cpp

```diff
@@ -256,6 +262,7 @@ DwarfScanner::blackListedDie(Dwarf_Die die, bool *dieBlackListed)
 			}
 		}
 	}
+#endif /* !defined(AIXPPC) */
```

Comment:
Nit: the convention is to use the same condition as is used in the `#if` directive (remove `!`).